packages/replay/src/debug.ts

```diff
@@ -1,62 +0,0 @@
-import dbg from "debug";
-import fs from "fs";
-import path from "path";
-import util from "util";
-import { getDirectory } from "./utils";
-
-const debugDebug = dbg("replay:cli:debug");
-
-const logDirPath = path.join(getDirectory(), "logs");
-export let logPath = path.join(
-  logDirPath,
-  "cli-" +
-    new Date()
-      .toISOString()
-      .replace(/:/g, "-")
-      .replace(/\.(\d+)Z$/, "-$1.log")
-);
-
-function init() {
-  try {
-    fs.mkdirSync(logDirPath, { recursive: true });
-  } catch (e) {
-    logPath = "";
-    debugDebug("Failed to create log directory %o", e);
-  }
-}
-
-let size = 0;
-export default function debug(namespace: string, pathToLog: string = logPath) {
-  size = Math.max(size, namespace.length);
-  const d = dbg(namespace);
-
-  if (process.env.REPLAY_CLI_DISABLE_LOG) {
-    return d;
-  }
```

Comment:
I didn't reimplement this one anywhere now. I wonder if there are any big concerns about it. It's not hard to add this but it's very much a "hidden feature" right now. Do we need it?